lib/toml-parser.js

```diff
@@ -162,9 +162,15 @@ function isList (obj) {
 }
 
 // in an eval, to let bundlers not slurp in a util proxy
-const utilInspect = eval(`require('util').inspect`)
-/* istanbul ignore next */
-const _inspect = (utilInspect && utilInspect.custom) || 'inspect'
+let _custom
+try {
+  const utilInspect = eval("require('util').inspect")
+  _custom = utilInspect.custom
+}
+catch (e) {
```

Comment:
The linter insists that this be on the same line as the close `}`. (I'll fix this myself when merging, if you don't get to it first.)